tests/fixtures/rendering.yml

```diff
@@ -1,3 +1,59 @@
+- array:
```

Comment:
We only have rendering tests that use the timestamp, in theory it should be possible to pass the individual variables and so I think we should test it as well.

Reply:
Done.